client/app.jsx

```diff
@@ -139,11 +149,12 @@ class App extends React.Component {
 
       //filter out all dates except the ones containing your stay
       while (true) {
-        let filtered = this.state.availableDates.filter(y => y.date.slice(0, -14) === dateString(selectedDay + ' ' + monthYear))
+        let filtered = this.state.availableDates.filter(y => y.date === dateString(selectedDay + ' ' + monthYear))
         if (allAvailable.indexOf(dateString(selectedDay + ' ' + monthYear)) !== -1) {
           //add selected class to dates and push to stay
           let padDay = _.padStart(selectedDay, 2, '0')
           let el = document.getElementById(padDay + ' ' + monthYear);
+          if (el === null) { break }
```

Comment:
Love this line. dont think the brackets are necessary tho? if (el === null) break; --> Not an important change lol. I just like writing one line conditional statements

Reply:
ohhh totally!  i'll get rid of them.  kinda looks awesome that way.